### DISCUSSION TITLE
Email part 'text/plain' not found

### DISCUSSION THREAD

**Discussion by brunaportugal** [bootstrap-email/bootstrap-email#193] [Help]
I updated the bootstrap-email gem from the `0.3.4` to the `1.1.6` version. Before, I was able to easily access the text/plain part by calling the `#text_part` method, and also to preview the text/plain emails without creating specific plain text templates for them. Since the update I can't seem to do that anymore. Am I missing something or is this functionality not there anymore?

Thank you!

**Comment by stuyam** [comment]
Plain text supported added in v1.3.0, reposting issue thread here: https://github.com/bootstrap-email/bootstrap-email/issues/195 Thank you for bringing this discussion to my attention, I think this is a great addition.
